### PR TITLE
add: `i3blocks-git`

### PR DESCRIPTION
**Pull request by laniusone**

### CODE REVIEW

**Review by Henryws — CHANGES_REQUESTED** [review]
image

**Comment by laniusone** [comment]
That's superweird, because I don't have `pkconf` installed, but it built properly on my system. Although it might be because I use zsh as user shell, so the check was skipped.

File: packages/i3blocks-git/i3blocks-git.pacscript [hunk 1]
```diff
@@ -0,0 +1,26 @@
+name="i3blocks-git"
+pkgname="i3blocks"
+pkgdir="/usr/src/pacstall/${name}"
```
Comment (Henryws) [inline packages/i3blocks-git/i3blocks-git.pacscript:3]:
```suggestion
pkgdir="${STOWDIR}/${name}"
```

**Review by Henryws — CHANGES_REQUESTED** [review]

File: packages/i3blocks-git/i3blocks-git.pacscript [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    git ls-remote "${url}" master | cut -f1 | cut -c1-8
+}
+version="$(pkgver)"
+
+prepare() {
+  ./autogen.sh && ./configure
+}
+
+build() {
+  make
```
Comment (Henryws) [inline packages/i3blocks-git/i3blocks-git.pacscript:20]:
```suggestion
  make -j"$(nproc)"
```

**Review by Henryws — CHANGES_REQUESTED** [review]

File: packages/i3blocks-git/i3blocks-git.pacscript [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  ./autogen.sh && ./configure
+}
+
+build() {
+  make -j"$(nproc)"
+}
+
+install() {
+  # Install i3blocks
+  sudo make install DESTDIR=$pkgdir
```
Comment (Henryws) [inline packages/i3blocks-git/i3blocks-git.pacscript:25]:
```suggestion
  sudo make install DESTDIR="$pkgdir"
```

**Review by Henryws — CHANGES_REQUESTED** [review]

File: packages/i3blocks-git/i3blocks-git.pacscript [hunk 4]
```diff
@@ -0,0 +1,26 @@
+name="i3blocks-git"
+pkgname="i3blocks"
+pkgdir="${STOWDIR}/${name}"
+description="A feed generator for text based status bars"
+url="https://github.com/vivien/${pkgname}.git"
+depends="autoconf gcc make pkgconf"
```
Comment (Henryws) [inline packages/i3blocks-git/i3blocks-git.pacscript:6]: These aren't dependencies that this program needs at runtime, these are only needed to compile right?

**Review by Henryws — CHANGES_REQUESTED** [review]

File: packages/i3blocks-git/i3blocks-git.pacscript (hunk 4 above)
Comment (laniusone) [reply packages/i3blocks-git/i3blocks-git.pacscript:6]: You're right, changing it to `build_depends`

**Review by Henryws — APPROVED** [review]

**Pull request merged**